modules/deploy/pages/deployment-option/cloud/manage-billing/gcp-commit.adoc

```diff
@@ -39,8 +39,6 @@ You will receive an email sent to the address you entered.
 This completes the registration and associates the email with a Redpanda account. 
 
 . On the **Accept your invitation to sign up** page, click **Sign up** or **Log in**. 
-+
```

Comment:
Why are we deleting this line? Without it the page ends abruptly.

Reply:
I can put it back replacing resource groups!